### PR TITLE
e2e: nightly 12/08 fixes

### PR DESCRIPTION
**Pull request by qoqobolo**
PR adds:
- closing the new ENS banner in several test groups
- `xfail` mark for a sticker test
- fix for the `Select token` button in Uniswap

### CODE REVIEW

File: test/appium/tests/critical/test_pairing_devices_sync.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from tests.base_test_case import MultipleSharedDeviceTestCase, create_shared_drivers
 from tests.users import transaction_senders, basic_user, ens_user, ens_user_message_sender
 from views.sign_in_view import SignInView
+from views.home_view import HomeView
```
Comment (pavloburykh) [inline test/appium/tests/critical/test_pairing_devices_sync.py]: is ens_banner_close_button unavailable without importing HomeView?

File: test/appium/tests/critical/test_pairing_devices_sync.py [hunk 2]
```diff
@@ -148,9 +149,15 @@ def prepare_devices(self):
         dapp = self.home_1.dapp_tab_button.click()
         web_page = dapp.open_url('status.im')
         web_page.add_to_bookmarks()
-        self.home_1.profile_button.double_click()
+
+        self.home_1.just_fyi("Close the ENS banner")
+        [home.home_button.double_click() for home in (self.home_1, self.home_2)]
+        for key in self.drivers:
```
Comment (pavloburykh) [inline test/appium/tests/critical/test_pairing_devices_sync.py]: What if to replace the lines 155, 156. 157 by [home.ens_banner_close_button.click() for home in (self.home_1, self.home_2)] , will it work?

File: test/appium/tests/critical/test_pairing_devices_sync.py (hunk 1 above)
Comment (qoqobolo) [reply test/appium/tests/critical/test_pairing_devices_sync.py]: yes, it's a new button that I just added to HomeView, so we need to import this view to the current file to use this button I suppose

Comment (pavloburykh) [reply test/appium/tests/critical/test_pairing_devices_sync.py]: > yes, it's a new button that I just added to HomeView, so we need to import this view to the current file to use this button I suppose

I supposed that if we use other buttons from HomeView then ens_banner_close_button should be available too...

File: test/appium/tests/critical/test_pairing_devices_sync.py (hunk 2 above)
Comment (qoqobolo) [reply test/appium/tests/critical/test_pairing_devices_sync.py]: Yes, it works. I just overcomplicated it for some reason :) will push the fix, thank you

File: test/appium/tests/critical/test_pairing_devices_sync.py (hunk 1 above)
Comment (qoqobolo) [reply test/appium/tests/critical/test_pairing_devices_sync.py]: got it. fixed too

File: test/appium/tests/critical/chats/test_1_1_public_chats.py [hunk 3]
```diff
@@ -978,6 +978,10 @@ def prepare_devices(self):
         self.home_2.add_contact(self.sender['public_key'])
         # To avoid activity centre for restored users
         [chat.send_message("hey!") for chat in (self.chat_1, self.chat_2)]
+
+        self.home_1.just_fyi("Close the ENS banner")
+        [home.home_button.double_click() for home in (self.home_1, self.home_2)]
+        [home.ens_banner_close_button.click() for home in (self.home_1, self.home_2)]
         [home.home_button.double_click() for home in (self.home_1, self.home_2)]
```
Comment (pavloburykh) [inline test/appium/tests/critical/chats/test_1_1_public_chats.py]: looks like redudant home_button.double_click() I guess line 985 can be deleted

**Review by pavloburykh — APPROVED** [review]

**Pull request merged**